### ISSUE TITLE
Accessibility: Editor content stuck scrolled to the right

### ISSUE THREAD

**Issue by jquesada2016** [microsoft/vscode#284320]
Type: <b>Bug</b>

Randomly, when editing code with very large text, the editor gets stuck scrolled to the far right. I can sometimes get it to align back to the left by opening and closing the terminal, or changing tabs, but often, this doesn't work. I have to keep trying this until it randomly decides to work again.

VS Code version: Code 1.107.1 (Universal) (994fd12f8d3a5aa16f17d42c041e5809167e845a, 2025-12-17T14:15:14.850Z)
OS version: Darwin arm64 25.1.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M4 Max (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: enabled_on<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|8, 4, 4|
|Memory (System)|128.00GB (67.54GB free)|
|Process Argv|--crash-reporter-id 9eef8a64-0777-4b98-8906-d2ab725c36fc|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (51)</summary>

Extension|Author (truncated)|Version
---|---|---
svelte-bundle|1Yi|1.0.0
monokai-charcoal-high-contrast|74t|3.5.0
slidev|ant|52.11.0
svelte-intellisense|ard|0.7.1
markdown-mermaid|bie|1.29.0
vscode-tailwindcss|bra|0.14.29
rescript-vscode|che|1.72.0
vscode-postgres|cko|1.4.3
dart-code|Dar|3.124.1
flutter|Dar|3.124.0
vscode-deno|den|3.47.0
vscode-wasm|dts|1.4.1
rust-syntax|dus|0.6.1
gitlens|eam|17.8.0
prettier-vscode|esb|11.0.2
syntax-highlighter|evg|0.5.0
dependi|fil|0.7.21
vscode-github-actions|git|0.28.2
gleam|gle|2.12.1
mysql-syntax|jak|1.3.1
rust-doc-viewer|JSc|4.2.0
edgedb|mag|0.2.5
vscode-scss|mrm|0.10.0
debugpy|ms-|2025.18.0
isort|ms-|2025.0.0
python|ms-|2025.20.1
vscode-pylance|ms-|2025.10.4
vscode-python-envs|ms-|1.14.0
remote-ssh|ms-|0.122.0
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.104.3
remote-explorer|ms-|0.5.0
vsliveshare|ms-|1.0.5959
vscode-purty|mva|0.6.0
vscode-python-typehint|njq|1.5.1
ide-purescript|nwo|0.26.6
language-purescript|nwo|0.2.10
rust-and-friends|nyx|1.0.0
material-icon-theme|PKi|5.29.0
rust-targets|Pol|1.2.1
vscode-thunder-client|ran|2.38.5
vscode-xml|red|0.29.0
vscode-yaml|red|1.19.1
rust-analyzer|rus|0.3.2719
surrealql|sur|0.2.16
svelte-vscode|sve|109.12.0
even-better-toml|tam|0.21.2
tauri-vscode|tau|0.2.9
vscode-nushell-lang|The|1.10.0
wgsl-analyzer|wgs|0.11.141
rust-mod-generator|Zha|1.0.12

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
pythonvspyt551cf:31249601
binariesv615:30325510
nativeloc1:31344060
dwcopilot:31170013
dwoutputs:31242946
copilot_t_ci:31333650
e5gg6876:31282496
pythonrdcb7:31342333
6518g693:31436602
aj953862:31281341
4f60g487:31327383
nes-set-on:31340697
6abeh943:31336334
cloudbuttont:31379625
todos-1:31405332
3efgi100_wstrepl:31403338
trigger-command-fix:31379601
use-responses-api:31390855
defaultusage:31403421
je187915:31401254
d5i5i512:31428709
ec5jj548:31422691
terminalsuggestenabled:31431119
cp_cls_c_966_ss:31435507
c0683394:31419495
478ah919:31426797
a5gib710:31434435
38bie571_auto:31426784
7a04d226_do_not_restore_last_panel_session:31438103
anthropic_thinking_t:31432745
31c7i317_no_change:31434515
cp_cls_c_1081:31433293

```

</details>

**Comment by jquesada2016** [comment]
No, that does not work. I always use that command for navigating to the start/end of the line. Also the accessibility view is always stuck on the right, no matter what I do. The cursor will go off the screen if I go to the start of the line on both the accessibility view and the editor.

**Comment by jquesada2016** [comment]
The only way I have found to reliably get the editor (not the accessibility view) to re-align is to click on the, what I think to be the scroll bar; the thing to the far left where there's like a heat map or something with warnings and errors. Here's a screenshot of what I mean. If I click on this, it will re-align the editor for a while until it starts happening again about an hour or so later.

<img width="5120" height="1440" alt="Image" src="https://github.com/user-attachments/assets/083d1f11-df1a-482c-91e3-ce760f7d250f" />
<img width="460" height="501" alt="Image" src="https://github.com/user-attachments/assets/31ba81ef-047d-43ab-920e-e7197e071822" />

**Comment by jquesada2016** [comment]
Ok, to add more repro context to this issue:

1. VoiceOver must be running
2. There must be multiple editor tabs open, like in my demo video
3. Each editor must have a lot of lines and there needs to be at least one line that is wider than the editor
4. it happens most when I use `F12` to go to definition
